crates/proof-of-sql/README.md

```diff
@@ -90,7 +90,7 @@ Parsing Query... 1.870256ms
 Generating Proof... 467.45371ms
 Verifying Proof... 7.106864ms
 Valid proof!
-Query result: OwnedTable { table: {Identifier { name: "b" }: VarChar(["hello", "world"])} }
+Query result: OwnedTable { table: {Ident { name: "b" }: VarChar(["hello", "world"])} }
```

Comment:
@varshith257 Please do run this example and put the results here. I don't think this is simply a string replacement here.